resources/views/pages-legacy/gameInfo.blade.php

```diff
@@ -278,7 +278,14 @@
 );
 ?>
 
-<?php if ($isFullyFeaturedGame): ?>
+@if (!$isFullyFeaturedGame)
```

Comment:
There's enough duplication here that I think it may be worth making it more concise. We can probably get away with just lifting it up into the PHP block directly above, as with this reduction we no longer need directives of any kind.

```php
sanitize_outputs(...);

$pageType = $isFullyFeaturedGame ? 'retroachievements:game' : 'retroachievements:hub';
$pageImage = media_asset($gameData['ImageIcon']);

if ($isFullyFeaturedGame) {
    $pageDescription = generateGameMetaDescription(
        $gameTitle,
        $consoleName,
        $numAchievements,
        $totalPossible,
        $isEventGame
    );
}
?>
```

Reply:
The psudocode that you provided is sound, added to latest.